data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -317,23 +362,7 @@ def load(self) -> None:
         # mkdir census
         self.OUTPUT_PATH.mkdir(parents=True, exist_ok=True)
 
-        columns_to_include = (
```

Comment:
I wanted to clean this up to be standard with our `self.COLUMNS_TO_KEEP` convention.

Reply:
As you said - consistency and parallelism :)